<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>2260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>3840</width>
    <height>2400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gamer’s Guide</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f0f0f2;
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>996</width>
           <height>2096</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>996</width>
           <height>2096</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>996</width>
           <height>2096</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 800 15pt &quot;JetBrains Mono NL&quot;;
color: rgb(0, 0, 0);
font-size: 20px;
</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Платформа&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 800 15pt &quot;JetBrains Mono NL&quot;;
color: rgb(0, 0, 0);
font-size: 20px;
</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Жанр&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="markBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: black
</string>
         </property>
         <item>
          <property name="text">
           <string>Все оценки</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8 - 10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4 - 7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 - 3</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="platformBox">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color:black
</string>
         </property>
         <item>
          <property name="text">
           <string>Все платформы</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Android</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>iOS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PlayStation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Xbox One</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 800 15pt &quot;JetBrains Mono NL&quot;;
color: rgb(0, 0, 0);
font-size: 20px;
</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Оценка&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="genresBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: black
</string>
         </property>
         <item>
          <property name="text">
           <string>Все жанры</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Экшен</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Стратегии</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Приключения</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Хорроры</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Спортивные</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>700</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="resetButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
color: red;</string>
         </property>
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
